src/views/Header/Header.js

```diff
@@ -34,7 +34,7 @@ type HeaderState = {
 };
 
 const APPBAR_HEIGHT = Platform.OS === 'ios' ? 44 : 56;
-const STATUSBAR_HEIGHT = Platform.OS === 'ios' ? 20 : 0;
+const STATUSBAR_HEIGHT = Platform.OS === 'ios' ? 20 : StatusBar.currentHeight;
```

Comment:
Please add a `|| 0` to the end to ensure that `STATUSBAR_HEIGHT` defaults to `0` instead of `undefined`. RN defines `StatusBar.currentHeight` as "Android only" and `react-navigation` runs on platforms that are not iOS or Android such as the web.

Reply:
Also this isn't actually correct.

On Platform.Version < 19 (api levels below 19) the relevant status bar height is always 0 because the ability to make status bars translucent was added in API level 19. (react-native supports API level 18).

Reply:
Which itself is also incorrect in this context. Because while Android introduced the ability to make status bars translucent in KitKat (API level 19) using an xml declaration ([the way Expo does it](https://github.com/expo/expo/blob/9a9dfb4103cb0cd5bfff8a97710350321655059f/android/app/src/main/res/values/styles.xml)) they didn't add the ability to make it translucent dynamically (the way `<StatusBar />` and most react-native apps that are translucent do it) until Lollipop (API level 21).

So if you are using Expo translucency doesn't apply `< 19`, if you are using `<StatusBar translucent />` it doesn't apply `< 21`.